### ISSUE TITLE
Tassel cap issue?

### ISSUE THREAD

**Issue by mmann1123** [jgrss/geowombat#71]
Just looking to plot a tcap band and got an unexpected error. 

``` python
fig, ax = plt.subplots(dpi=150)

with gw.config.update(sensor='qb', scale_factor=0.0001):
    with gw.open(rgbn) as ds:
        tcap = ds.gw.tasseled_cap()
        tcap.sel(band='wetness').plot(robust=True, ax=ax)
plt.tight_layout(pad=1)
```
throws
```
TypeError: Plotting requires coordinates to be numeric, boolean, or dates of type numpy.datetime64, datetime.datetime, cftime.datetime or pandas.Interval. Received data of type object instead.
```

**Comment by jgrss** [comment]
Should be fixed in [master@37bd27ba](https://github.com/jgrss/geowombat/commit/37bd27baa6434e9525acfceba8c6b53325f3f1db).

**Comment by mmann1123** [comment]
Solved thanks
